### PR TITLE
NixOS: Plasma 5 tests and warnings

### PR DESCRIPTION
**Pull request by ttuegel**
### Motivation for this change

- The Plasma test was not working since the KDE overhaul.
- Building the graphical ISO produced an avoidable warning.

### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/tests/plasma5.nix [hunk 1]
```diff
@@ -17,7 +17,8 @@ import ./make-test.nix ({ pkgs, ...} :
         user = "alice";
       };
     };
-    services.xserver.desktopManager.kde5.enable = true;
+    services.xserver.desktopManager.plasma5.enable = true;
+    services.xserver.desktopManager.default = "plasma5";
```
Comment (grahamc) [inline nixos/tests/plasma5.nix]: Does this mean users need to put this in their configuration now?

Comment (grahamc) [inline nixos/tests/plasma5.nix]: (the `default = "plasma5"` bit)

Comment (grahamc) [inline nixos/tests/plasma5.nix]: This might be related: 4b6f021251e60af9f61311e116eef043a2ed0d5d

Comment (ttuegel) [inline nixos/tests/plasma5.nix]: If they use the auto-login feature, yes. I don't know why it is suddenly necessary.

**Pull request merged**